Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphProjection.php

```diff
@@ -92,7 +109,26 @@ public function __construct(
 
     public function setUp(): void
     {
-        foreach ($this->determineRequiredSqlStatements() as $statement) {
+        $statements = $this->determineRequiredSqlStatements();
+
+        // MIGRATION from 2024-05-25: copy data from "cr_<crid>_p_workspace"/"cr_<crid>_p_contentstream" to "cr_<crid>_p_graph_workspace"/"cr_<crid>_p_graph_contentstream" tables
+        $legacyWorkspaceTableName = str_replace('_p_graph_workspace', '_p_workspace', $this->tableNames->workspace());
+        if (
+            $this->dbal->getSchemaManager()->tablesExist([$legacyWorkspaceTableName])
+            && !$this->dbal->getSchemaManager()->tablesExist([$this->tableNames->workspace()])
+        ) {
+            $statements[] = 'INSERT INTO ' . $this->tableNames->workspace() . ' (name, baseWorkspaceName, currentContentStreamId, status) SELECT workspaceName AS name, baseWorkspaceName, currentContentStreamId, status FROM ' . $legacyWorkspaceTableName;
```

Comment:
does the casing not matter? In the legacy projection it was `currentcontentstreamid` ... just wondering about the select

Reply:
this is a good question:

> Partition, subpartition, column, index, stored routine, event, and resource group names are not case-sensitive on any platform, nor are column aliases

So this should be fine, however table names _could_ be depending on platform so there we need to make sure it matches everywhere.

Reply:
still i adjusted it, and also double checked that we dont use the old column names anymore (where we renamed them)